components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -375,7 +375,7 @@ func (r *OpenshiftNotebookReconciler) UnsetNotebookCertConfig(notebook *nbv1.Not
 	log := r.Log.WithValues("notebook", notebook.Name, "namespace", notebook.Namespace)
 
 	// Get the notebook object
-	envVars := []string{"PIP_CERT", "REQUESTS_CA_BUNDLE", "SSL_CERT_FILE", "PIPELINES_SSL_SA_CERTS"}
+	envVars := []string{"PIP_CERT", "REQUESTS_CA_BUNDLE", "SSL_CERT_FILE", "PIPELINES_SSL_SA_CERTS", "GIT_SSL_CAINFO"}
```

Comment:
How is this variable different from `GIT_SSL_CAPATH`. Looking at https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpsslCAInfo, I can't make sense of it.

Reply:
Aha, got it, one is to provide a single pem file and the other to provide directory of pem files.